bundle.Dockerfile

```diff
@@ -18,3 +18,4 @@ LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
 COPY bundle/manifests /manifests/
 COPY bundle/metadata /metadata/
 COPY bundle/tests/scorecard /tests/scorecard/
+LABEL com.redhat.openshift.versions="v4.11-v4.14"
```

Comment:
Is there a way to omit the upper version? If no we need to document updating this during the release.

Reply:
It seems it needs to be added :/

Reply:
Also it would be good to bump this one to 4.15 to make sure the operator lands on the next OCP version that is going to be released soon